src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -140,10 +140,6 @@
 	<PackageReference Include="SharpDX.Mathematics" Version="4.2.0" />
     <Reference Include="Dynamo.Microsoft.Xaml.Behaviors">
       <HintPath>..\..\extern\Microsoft.Xaml.Behaviors\$(TargetFramework)\Dynamo.Microsoft.Xaml.Behaviors.dll</HintPath>
-    </Reference>
-    <PackageReference Include="Prism.Core" Version="8.1.97" Condition=" '$(TargetFramework)' != 'net48' " />
```

Comment:
I think you should have left <PackageReference Include="Prism.Core" Version="8.1.97" Condition=" '$(TargetFramework)' != 'net48' " /> in there
Maybe just remove the condition attribute (ex <PackageReference Include="Prism.Core" Version="8.1.97"/>)

Reply:
I removed it since it is already referenced in the line below it.
https://github.com/DynamoDS/Dynamo/pull/14209/files/e23d79094af0f648d893c2a3fcb07c9c47627ba9..efebe65238c7dee671845e6834945248f5828120#diff-3a61f1c3321e4a236dbd60644db81ea7d71eb5eb709191187126610ff76abd09R145